recipes/rivgraph/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "rivgraph" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/veinsoftheearth/rivgraph/archive/v{{ version }}.tar.gz
+  sha256: 797b3dbba2a2f135a39a9cbaebdf59f9194fa48808211c5733dee56b510a505c
+  # sha256 is the preferred checksum -- you can get it for a file with:
+  #  `openssl sha256 <file name>`.  
+
+build:
+  noarch: python
+  number: 0  
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:  
+  host:
+    - pip
+    - python >=3.7    
+    - geopandas
+    - scikit-image
+    - opencv
+    - networkx <2.7 # necessary for correct numpy ops
+    - shapely =1.7.1
+    - matplotlib-base
+    - fastdtw
+    - loguru
+    
```

Comment:
Nominally these should only bee needed in `run` (unless the package is importing itself which shouldn't be needed).

```suggestion
```

Reply:
Omitting these from the host section was giving me build errors. Is self importing going on in https://github.com/VeinsOfTheEarth/RivGraph/blob/master/rivgraph/__init__.py#L3 ?
@elbeejay @jonschwenk

Reply:
The issue is this line:

* https://github.com/VeinsOfTheEarth/RivGraph/blob/fb3baecd2a7a5c0e15b6771f1d757a59219bbf53/setup.py#L2

There are several ways of avoiding it which are summarised here:

* https://packaging.python.org/en/latest/guides/single-sourcing-package-version/

My personal favourite is `setuptools_scm` (+`importlib.metadata` in `__init__.py` to set the `__version__` variable).

Regardless it doesn't matter for conda-forge so LGTM 👍